tests/collections/asr/numba/rnnt_loss/test_rnnt_pytorch.py

```diff
@@ -0,0 +1,247 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Suggest to drop the folder named numba as it is not the main topic and may add future features based on numba later. You may have a folder named "rnnt", then you can have numba_rnnt under that. Both are OK, your choice.

Reply:
Hmm my original plan was to make numba entirely optional, and so importing the numba section would silently be disabled until at runtime someone requested a numba feature.

In the end, turns out thats not really realistic to implement since so much numba code is being used. 

On the other hand, having numba dir itself is valuable - so as to separate numba implementations vs normal asr parts (implemented in pytorch). I think lets keep it for now, and later if we get more numba stuff, refactor into `commons` or remove numba dir entirely.